frontends/ol-components/src/components/LearningResourceExpanded/LearningResourceExpandedV2.tsx

```diff
@@ -684,8 +690,20 @@ const LearningResourceExpandedV2: React.FC<LearningResourceExpandedV2Props> = ({
   onAddToUserListClick,
   closeDrawer,
 }) => {
+  const outerContainerRef = useRef<HTMLDivElement>(null)
+  const searchParams = useSearchParams()
+  const searchParamsRef = useRef(searchParams)
+  useEffect(() => {
```

Comment:
Would this work instead?
```ts
  const outerContainerRef = useRef<HTMLDivElement>(null)
  useEffect(() => {
    if (outerContainerRef.current) {
      outerContainerRef.current.scrollTo(0, 0)
    }
  }, [resource?.id])
```
Only possible quirk i see is that if you scroll while the resource is still loading, it will scroll up when when the resource finishes loading.

If that seems undesirable, maybe pass the ID into v2expanded as a prop. (Alternatively the effect could run when `searchParams.get("resource")` changes, and eliminate the search params ref, but currently v2expanded has no knowledge of routing, e.g., it doesn't know the "resource" querykey is involved at all, mildly nice to presreve that..)